logger/logger.go

```diff
@@ -0,0 +1,33 @@
+package logger
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const LOG_FILE_NAME = "harbourbridge.log"
+
+var Log *zap.Logger
+
+func InitializeLogger(inputLogLevel string) error {
```

Comment:
Should we add some case sensitivity handling over here?

Reply:
The `zap` library takes care of case sensitivity implicitly. For example, `INFO` and `info` are both translated to the `INFO` log level.